frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -104,9 +135,184 @@ describe("WMG tissue auto-expand", () => {
+   * filter for B Cell. Only B Cell should appear and Blood should be expanded.
+   */
+  test("Tissue auto expansion - cross filter with Publication filter selected", async ({
+    page,
+  }) => {
+    const tissues = ["blood"];
+    await loadPageAndTissues(page);
+    await filterSelection(
+      page,
+      PUBLICATION_FILTER_TEST_ID,
+      PUBLICATION_FILTER_LABEL,
+      PUBLICATION_FILTER_SELECTION[0]
+    );
+    await filterTissues(page, tissues);
+    await collapseTissue(page, tissues[0]);
+    await filterCellType(page, 1);
+    await checkTissues(page, tissues);
+    await checkElementVisible(page, EXPECTED_VISIBLE_CELL, CELL_TYPE_TEST_ID);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Publication filter, check expansion
+   * Filter 3 Tissues ["abdomen", "axilla", "blood"]. Collapse Blood. Filter
+   * 'Ahren et al. Neuron 2021' from the Publication filter. Only Blood should remain
+   * visible and collapsed. Add Cell filter for B Cell. Only B Cell should be
+   * visible under expanded Blood tissue node
+   */
+  test("Tissue auto expansion - cross filter with Publication filter, check expansion", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await filterTissues(page);
+    await collapseTissue(page, "blood");
+    await filterSelection(
+      page,
+      PUBLICATION_FILTER_TEST_ID,
+      PUBLICATION_FILTER_LABEL,
+      PUBLICATION_FILTER_SELECTION[1]
+    );
+    await expect(page.getByTestId(TISSUE_NODE_TEST_ID)).toHaveCount(1);
+    await checkTissues(page, ["blood"], []);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Self-Reported Ethnicity filter
+   * Expand Breast. Select African from Ethnicity filter. Breast should remain expanded.
+   * Add Nose to the Tissue filter. Nose should appear expanded.
+   */
+  test("Tissue auto expansion - cross filter with Self-Reported Ethnicity filter", async ({
+    page,
+  }) => {
+    await loadPageAndTissues(page);
+    await expandTissue(page, SELF_REPORTED_ETHNICITY_TISSUE[0]);
+    await filterSelection(
+      page,
+      SELF_REPORTED_ETHNICITY_FILTER_TEST_ID,
+      SELF_REPORTED_ETHNICITY_FILTER_LABEL,
+      SELF_REPORTED_ETHNICITY_FILTER_SELECTION
+    );
+    await filterTissues(page, SELF_REPORTED_ETHNICITY_TISSUE);
+    await checkTissues(page, SELF_REPORTED_ETHNICITY_TISSUE);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Disease filter
+   * Filter 3 Tissues ["abdomen", "axilla", "brain"]. From the Disease filter select influenza.
+   * Blood should appear only and expanded. Add cell type filter for B Cell. Only B Cell should
+   * appear under Blood and expanded. Remove influenza. Blood should remain expanded and cell count
+   * increase.
+   */
+  test("Tissue auto expansion - cross filter with Disease filter", async ({
+    page,
+  }) => {
+    const tissues = ["blood"];
+    await loadPageAndTissues(page);
+    await filterTissues(page);
```

Comment:
here we are filtering 3 tissues... that's what happens when we mix constants with hardcoding lol it get's confusing

Reply:
Ahh I see! Forgot about the default tissue filters come with `filterTissues()` 😆 My bad! 

Maybe use this instead of `tissue` would help?

```ts
const EXPECTED_FILTERED_TISSUES_WITH_DISEASE_FILTER = ["blood"];

test("Tissue auto expansion - cross filter with Disease filter", async ({
    page,
  }) => {
  await loadPageAndTissues(page);
  ...
  await checkTissues(page, EXPECTED_FILTERED_TISSUES_WITH_DISEASE_FILTER);
});
```